gd1_helpers/membership/gd1_model.py

```diff
@@ -8,67 +8,81 @@
     Normal1DSplineMixtureVariable,
     Normal1DSplineVariable,
     UniformVariable,
+    Normal1DVariable
 )
 
-phi1_lim = (-100, 20)
+#phi1_lim = (-100, 20)
 
+class Base:
+    phi1_lim = (-20, 20)
+    coord_bounds = {"phi1": phi1_lim, "phi2": (-7,7), "pm1": (-20,20), "pm2": (-20,20)}
 
-class GD1Base:
-    coord_names = ("phi2", "pm1", "pm2")
-    coord_bounds = {"phi1": phi1_lim, "phi2": (-7, 5), "pm1": (-15, -1.0)}
+    @classmethod
+    def setup(cls, pawprint, data):
+        cls.pawprint, cls.data = pawprint, data
 
-    default_grids = {
-        "phi1": np.arange(*coord_bounds["phi1"], 0.2),
-        "phi2": np.arange(*coord_bounds["phi2"], 0.1),
-        "pm1": np.arange(*coord_bounds["pm1"], 0.1),
-        "pm2": np.arange(-10, 10 + 1e-3, 0.1),
-    }
+        coord_names = ("phi2", "pm1", "pm2")
```

Comment:
Why move this inside?

Reply:
Hmm, I don't think there was a reason. Probably better to move back out